src/Device/AndroidSensors.cpp

```diff
@@ -287,6 +288,49 @@ DeviceDescriptor::OnHeartRateSensor(unsigned bpm) noexcept
   e.Commit();
 }
 
+void
+DeviceDescriptor::OnEngineSensors(uint16_t cht_temp,
+                                      uint16_t egt_temp,
+                                      uint16_t oat_temp,
+                                      uint32_t atmospheric_pressure,
+                                      uint16_t revs_per_sec) noexcept
```

Comment:
Indentation is still broken and you're still using the wrong types

Reply:
What's wrong with the types? Should be valid from c++11. How would you like the types to be?

Reply:
Temperature, AtmosphericPressure - your choice of plain integers is error prone because it's not type-safe. What's the unit for the temperature? What's the unit for pressure? Using the correct types makes the code more robust while not adding any bit of overhead.